content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -16,24 +16,108 @@ your model training using Iterative Studio.
 > to your ML project repositories, you can follow the instructions given below
 > to run experiments directly from Iterative Studio.
 
+## Use the Iterative Studio wizard to set up your CI action
+
+When you select one of the commits in your project table and click on the `Run`
+button, an input form will open. At the top of this form, you will see a small
+message that invites you to set up your CI in case you have not done it yet.
+
+![](https://static.iterative.ai/img/studio/set_up_cml_message.png)
+
+Click on `Click here` to use the wizard to create your CML workflow within your
+CI/CD vendor.
+
+The wizard UI is divided in two sections:
+
+- Left section with 2 sets of parameters:
+
+  1. [Configuration of your self-hosted runner, which is used in the `Deploy runner` step of your CI process](#step-1-deploy-runner)
+  2. [Model training script, which is used in the `Train` step of your CI process](#step-2-train)
+
+- Right section which displays in real-time the
+  [generated yaml to be used in your CI set up](#ci-yaml).
+
+By default, some standard values are assumed for all the inputs and a complete
+workflow (based on these default values) is ready for you to copy.
+
+![](https://static.iterative.ai/img/studio/set_up_cml_full.png)
+
+### Step 1: Deploy runner
+
+This step is responsible for launching a self-hosted runner within your cloud
+vendor. The parameters listed here are a subset of the parameters for
+[CML self-hosted runners](https://cml.dev/doc/self-hosted-runners).
+
+- `Spot`: Whether you want to launch a spot cloud instance, cutting down the
+  costs of your training.
+
+- `Reuse`: Values for the CML flags `reuse` and `reuse-idle`. See all
+  [CML options](https://cml.dev/doc/ref/runner#options) for details.
+
+- `Labels`: Text labels to identify your CML runners from other self hosted
+  runners that you might have.
+
+- `Cloud`: Your cloud provider.
+
+- `Region`: Cloud-vendor specific region or a CML synthetic region (an
+  abstraction across all the cloud vendors).
+
+- `Type`: Cloud-vendor specific instance type or a CML synthetic type
+  `M`/`L`/`XL` (an abstraction across all the cloud vendors).
+
+  `Type` is also tied to GPU behavior. If you choose an instance with a
+  selectable GPU (such as a CML instance type or any GCP instance), the `GPU`
+  parameter will show up.
+
+- `HDD size`: Hard disk size in GB. We highly recommend you to enter a big
+  enough value (eg, 100) to avoid unexpected runner termination due to hard disk
+  exhaustion.
+
+### Step 2: Train
+
+This is the script needed for your runner to train your model. The default
+template is a very common combination of CML and DVC taking into account that
+DVC enables you to make the most of Iterative Studio. You can update this script
+to reflect your exact model training process, whether you use DVC or not.
+
+### Generated CI yaml
+
+The right section displays the generated CI yaml content, which reflects all
```

Comment:
This whole section could be skipped IMO. We already mentioned it and said it's for copying. Describing every button is unnecessary (Studio is a visual product -- should be self-documenting I think).

Reply:
Done. PTAL.